Services/CO.CDP.Forms.WebApi/UseCase/GetFormSectionQuestionsUseCase.cs

```diff
@@ -16,13 +16,18 @@ public class GetFormSectionQuestionsUseCase(IFormRepository formRepository, IMap
         if (section == null)
             return null;
 
-        var answerSet = await formRepository.GetFormAnswerSetsAsync(sectionId, organisationId);
+        var answerSets = await formRepository.GetFormAnswerSetsAsync(sectionId, organisationId);
+        if (answerSets != null)
+        {
+            var currentSharedConsent = answerSets.Select(x => x.SharedConsent).OrderByDescending(x => x.UpdatedOn).FirstOrDefault();
+            answerSets = answerSets.Where(x => x.SharedConsent == currentSharedConsent).ToList();
+        }
```

Comment:
Do we know why the `GetFormAnswerSetsAsync` query does not return what we need?

Reply:
done.